pkg/collectors/subsystem.go

```diff
@@ -198,27 +196,23 @@ func (f *PerfCollector) collectSystemMetrics(ch chan<- prometheus.Metric, fsRest
 	newSystemMetrics(ch, f.sysInfoDescriptors[SystemMetadata], 0, &systemInfo)
 
 	// [lssystem]: physical_capacity
-	PhysicalTotalCapacityB := sysInfoResults[PhysicalTotalCapacity].(string)
-	PhysicalTotalCapacityIB := convertStringToBytes(PhysicalTotalCapacityB)
-	physicalTotalCapacityFormat, err := humanize.ParseBytes(PhysicalTotalCapacityIB)
+	physicalTotalCapacity, err := strconv.ParseFloat(sysInfoResults[PhysicalTotalCapacity].(string), 64)
 	if err != nil {
 		log.Errorf("get physical capacity failed: %s", err)
 	}
-	newSystemCapacityMetrics(ch, f.sysCapacityDescriptors[SystemPhysicalTotalCapacity], float64(physicalTotalCapacityFormat), &systemName)
+	newSystemCapacityMetrics(ch, f.sysCapacityDescriptors[SystemPhysicalTotalCapacity], physicalTotalCapacity, &systemName)
 
 	// [lssystem]: physical_free_capacity
-	physicalFreeCapacityB := sysInfoResults[PhysicalFreeCapacity].(string)
-	physicalFreeCapacityIB := convertStringToBytes(physicalFreeCapacityB)
-	physicalFreeCapacityFormat, err := humanize.ParseBytes(physicalFreeCapacityIB)
+	physicalFreeCapacity, err := strconv.ParseFloat(sysInfoResults[PhysicalFreeCapacity].(string), 64)
 	if err != nil {
 		log.Errorf("get physical capacity failed: %s", err)
 	}
-	newSystemCapacityMetrics(ch, f.sysCapacityDescriptors[SystemPhysicalFreeCapacity], float64(physicalFreeCapacityFormat), &systemName)
+	newSystemCapacityMetrics(ch, f.sysCapacityDescriptors[SystemPhysicalFreeCapacity], physicalFreeCapacity, &systemName)
 
 	// used = total - free
-	physicalUsedCapacity := physicalTotalCapacityFormat - physicalFreeCapacityFormat
-	newSystemCapacityMetrics(ch, f.sysCapacityDescriptors[SystemPhysicalUsedCapacity], float64(physicalUsedCapacity), &systemName)
-	log.Infof("system capacity total: %d, free: %d, used: %d", physicalTotalCapacityFormat, physicalFreeCapacityFormat, physicalUsedCapacity)
+	physicalUsedCapacity := physicalTotalCapacity - physicalFreeCapacity
+	newSystemCapacityMetrics(ch, f.sysCapacityDescriptors[SystemPhysicalUsedCapacity], physicalUsedCapacity, &systemName)
+	log.Infof("system capacity total: %v, free: %v, used: %v", physicalTotalCapacity, physicalFreeCapacity, physicalUsedCapacity)
```

Comment:
Why %v and not %d?

Reply:
because when it was %d I had compilation errors- 
"pkg/collectors/subsystem.go:215: Infof format %d has arg physicalTotalCapacity of wrong type float64"

Reply:
AFAIK - %d is for integers, %f / %e is for floats , %v is a general value

Reply:
so maybe %f is better?

Reply:
fixed